src/main/kotlin/src/racingcar/domain/Winner.kt

```diff
@@ -0,0 +1,10 @@
+package src.racingcar.domain
+
+class Winner(
+    private val racingCars: List<Car>
+) {
+    fun findWinners(): List<Car> {
+        val max = racingCars.maxOf { it.moveCount }
+        return racingCars.filter { car -> car.moveCount >= max }
```

Comment:
```suggestion
        return racingCars.filter { car -> car.moveCount == max }
```

`==` 비교로도 충분하지 않을까요?